MapboxNavigation/RouteVoiceController.swift

```diff
@@ -159,9 +159,12 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     }
     
     func duckAudio() throws {
-        let categoryOptions: AVAudioSessionCategoryOptions = [.duckOthers, .interruptSpokenAudioAndMixWithOthers]
-        try AVAudioSession.sharedInstance().setMode(AVAudioSessionModeSpokenAudio)
-        try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: categoryOptions)
+        if #available(iOS 12.0, *) {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, mode: AVAudioSessionModeVoicePrompt, options: [.duckOthers])
+        } else {
+            try AVAudioSession.sharedInstance().setMode(AVAudioSessionModeSpokenAudio)
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSessionCategoryPlayback, with: [.duckOthers])
```

Comment:
Sorry for the back-and-forth: from https://github.com/mapbox/mapbox-navigation-ios/pull/1933#discussion_r248424392, I gather that `[.duckOthers, .interruptSpokenAudioAndMixWithOthers]` is correct in conjunction with either `AVAudioSessionModeSpokenAudio` or `AVAudioSessionModeVoicePrompt`, since we’re using this audio session for short text-to-speech prompts as described in the `AVAudioSessionModeVoicePrompt` documentation, as opposed to for long-form audiobook reading as described in the `AVAudioSessionModeSpokenAudio` documentation.

Reply:
@1ec5 resolved in the recent commit ✅